spec.bs

```diff
@@ -218,6 +218,18 @@ payment service providers are likely to build them on behalf of merchants.
 NOTE: Relying Parties that wish to provide the authentication user experience
 may still do so using SPC from an iframe.
 
+### Cryptographic evidence of device binding ### {#sctn-use-case-device-binding}
```

Comment:
@ianbjacobs - would love your thoughts, on this PR in general but particularly on the non-normative sections where it speaks to the motivation + needs